.gitignore

```diff
@@ -175,5 +175,3 @@ src/autorevision.h
 # someone seems to keep personal notes in this file
 .idea
 
-Pods
-Podfile.lock
```

Comment:
You probably need to re-run a "pod install" or a "pod update" to get back to a consistent state after this change.

Reply:
Yes that´s right. So about that change: If the Pods directory and the Podfile.lock is not excluded, it should be in the repository, right? Problem is, they are not. So either we keep those two files excluded, or we really sync the installed pods and the podfile.lock. The way it is now is no good practice and leads to problems inevitably. Actually it was the first major hurdle to contribute to Vienna for me, because Cocoapods kept complaining.

Reply:
Hmm they are in the repository though... or am I missing something?

Reply:
No, some Pods' target support files went missing from the repo on commit f082eadbb6ecdd747d4ea90df7286fa7d7ce955a
So I cannot build with current state of @TAKeanice's branch : I have to perform a `pod install` first to recover the missing files.

My [personal branch](https://github.com/barijaona/vienna-rss/commits/tab_bar3) seems OK after adding the latest commit (0d649ec947424459d556da79980bd1bbf31f08e2) correcting the copy frameworks sequence.

@TAKeanice, if you don't mind, you can clone my branch and submit an alternative pull request to this one. Having been cherry-picked without all these pod changes complexities, mine is easier to read. It is also on par with current master.

Reply:
ahh i see.... i will pull tonight and have a proper review :)

Reply:
@josh64x2 you are right, I somehow missed that they are there. Must have been something different with Cocoapods when I first tried to build Vienna then. I have already added the copy frameworks step in my most recent commit. I will also re-add the target support files and merge the changes from the current master, then my branch should be fine again.